presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -411,6 +411,50 @@ public void testLeastWithNaN()
         functionAssertions.tryEvaluate("least(1.5, 0.0 / 0.0)");
     }
 
+    @Test
+    public void testToBase()
+            throws Exception
+    {
+        assertFunction("to_base(2147483648, 16)", "80000000");
+        assertFunction("to_base(255, 2)", "11111111");
+        assertFunction("to_base(-2147483647, 16)", "-7fffffff");
+    }
+
+    @Test(expectedExceptions = PrestoException.class, expectedExceptionsMessageRegExp = "\\QRadix must be in [2, 36]: 1\\E")
+    public void testToBaseWithInvalidBase()
+    {
+        functionAssertions.tryEvaluate("to_base(255, 1)");
```

Comment:
Copy `assertInvalidFunction` from one of the other test classes like `TestStringFunctions`

(we will consolidate this into a base class later)